README.md

```diff
@@ -17,6 +17,7 @@ To get the database and tests running:
 
 1. Start PostgreSQL. You will need to be able to create databases; set the
    `DB_`* variables in your environment if needed.
+1. Install [PhantomJS](http://phantomjs.org/download.html) and have it in your PATH
```

Comment:
I think they are pretty different instructions for installing with different OSes. Do you think it's worth the extra logic?